### ISSUE TITLE
[BUG] Scroll does not work when default zoom

### ISSUE THREAD

**Issue by aalberrty** [EhPanda-Team/EhPanda#406]
**Describe the bug**
Scrolling is disabled when zoom is default, after zoomed in it works but when zoomed back out to default it stops working again.

**To Reproduce**
1. Open any gallery
2. Scrolling does not work as it should

**Expected behavior**
Scrolling should work under both default zoom and when zoomed in.

**Please complete the following information:**
 - Device: iPhone 16 Pro Max
 - OS: iOS 16.2
 - App Version 2.7.9

**Additional context**
This bug should be caused by commit #405 where commit cf4f6ed changed the gesture handling logic.

**Comment by aalberrty** [comment]
> same problem. do you have any way to solve this?

this was fixed with [#407 ](https://github.com/EhPanda-Team/EhPanda/pull/407) for iOS 16, are you on iOS 26? I am experiencing the same issue with iOS26.

**Comment by J1npen** [comment]
> > same problem. do you have any way to solve this?
> 
> this was fixed with [#407 ](https://github.com/EhPanda-Team/EhPanda/pull/407) for iOS 16, are you on iOS 26? I am experiencing the same issue with iOS26.

Yes! Same problem! After updating to iOS26, I suddenly can't scroll anymore! 😧
Now I think the only solution is to wait for developer to fix this bug. But however, he hasn't maintained this project for a long time. 😢

**Comment by aalberrty** [comment]
> > > same problem. do you have any way to solve this?
> > 
> > 
> > this was fixed with [#407 ](https://github.com/EhPanda-Team/EhPanda/pull/407) for iOS 16, are you on iOS 26? I am experiencing the same issue with iOS26.
> 
> Yes! Same problem! After updating to iOS26, I suddenly can't scroll anymore! 😧 Now I think the only solution is to wait for developer to fix this bug. But however, he hasn't maintained this project for a long time. 😢

i published a fix in my fork, should work fine now [here](https://github.com/aalberrty/EhPanda/releases/tag/2.8)

**Comment by J1npen** [comment]
> > > > same problem. do you have any way to solve this?
> > > 
> > > 
> > > this was fixed with [#407 ](https://github.com/EhPanda-Team/EhPanda/pull/407) for iOS 16, are you on iOS 26? I am experiencing the same issue with iOS26.
> > 
> > Yes! Same problem! After updating to iOS26, I suddenly can't scroll anymore! 😧 Now I think the only solution is to wait for developer to fix this bug. But however, he hasn't maintained this project for a long time. 😢
> 
> i published a fix in my fork, should work fine now [here](https://github.com/aalberrty/EhPanda/releases/tag/2.8)

Oh my god, you’re amazing! I know how to fix the errors in the code, but I don’t know how to create an IPA file for installation. How did you do it?

**Comment by aalberrty** [comment]
> > > > > same problem. do you have any way to solve this?
> > > > 
> > > > 
> > > > this was fixed with [#407 ](https://github.com/EhPanda-Team/EhPanda/pull/407) for iOS 16, are you on iOS 26? I am experiencing the same issue with iOS26.
> > > 
> > > 
> > > Yes! Same problem! After updating to iOS26, I suddenly can't scroll anymore! 😧 Now I think the only solution is to wait for developer to fix this bug. But however, he hasn't maintained this project for a long time. 😢
> > 
> > 
> > i published a fix in my fork, should work fine now [here](https://github.com/aalberrty/EhPanda/releases/tag/2.8)
> 
> Oh my god, you’re amazing! I know how to fix the errors in the code, but I don’t know how to create an IPA file for installation. How did you do it?

there is a lot of ways, main repo used actions to compile, i compiled it locally
